iron-form.html

```diff
@@ -128,10 +135,12 @@
 
       // Holds all the custom elements registered with this form.
       this._customElements = [];
+      // Holds the initial values of the custom elements registered with this form.
+      this._customElementsInitialValues = [];
```

Comment:
Since we will soon have `Polymer.dom(...).observeNodes`, I think it might be good to file an issue for updating these arrays based on lightDOM tree changes. WDYT?